controllers/categories-controllers.js

```diff
@@ -13,26 +9,3 @@ exports.getCategories = (req, res, next) => {
       next(err);
     });
 };
-
```

Comment:
I was a little worries, but good job at extracting this out into a new controllers file 👍